src/compiler.adoc

```diff
@@ -962,7 +962,7 @@ test directory:
 This new script will compile and watch both directories "src" and "test", and
 sets the new entry point to the `appfortesting.core_tests` namespace.
 
-And then, write some test code into the `core_tests.cljs` file:
+Next, write some test code into the `core_tests.cljs` file:
```

Comment:
since we took an explanatory detour in the preceding paragraph, I think "next" fits better.